src/engine.c

```diff
@@ -881,6 +881,24 @@ ENGINE_getKeyState(ENGINE* engine, char* keyName) {
   return state[scancode];
 }
 
+internal uint32_t
+ENGINE_findMouseCursorIndex(ENGINE* engine, char* cursorName) {
+  int cursorIndex;
+  for (int index = 0; index < 12; index++) {
```

Comment:
SDL Enums usually have an entry at the end which can be used as the upper limit of entries. See here: https://github.com/libsdl-org/SDL/blob/e18be04bc6ae964beb175b5468595d640a100eec/include/SDL_mouse.h#L60

So we should use `SDL_NUM_SYSTEM_CURSORS` instead of 12. :)

Reply:
fixed in 8d7232c